server/services/reportedAdjudicationsService.ts

```diff
@@ -47,6 +47,22 @@ export default class ReportedAdjudicationsService {
     }
   }
 
+  async getSimplifiedReportedAdjudication(adjudicationNumber: number, user: User): Promise<ConfirmedOnReportData> {
```

Comment:
I really don't like this name - it would be better to call it something like `getConfirmationDetails` and maybe rename the other one to indicate it returns other information for further instructions - `getDetailsForInstruction` (I can't think of a good one right now)

There is no need to add ReportedAdjudication as that is part of the service name